assets/src/edit-story/components/panels/layer/layer.js

```diff
@@ -80,10 +90,28 @@ const LayerDescription = styled.div`
   color: ${({ theme }) => theme.colors.fg.v1};
 `;
 
+const LockedIcon = styled(Locked)`
+  height: 18px !important;
+  width: 18px !important;
+`;
+
+const BackgroundDescription = styled.div`
+  opacity: 0.5;
+`;
+
+const LayerContentContainer = styled.div`
+  margin-right: 8px;
+`;
+
 function Layer({ layer }) {
   const { LayerIcon, LayerContent } = getDefinitionForType(layer.type);
   const { isSelected, handleClick } = useLayerSelection(layer);
   const { handleStartReordering } = useLayerReordering(layer);
+  const {
```

Comment:
Hmm. I'm seeing the background showing up in the layer panel now, even when not specified. Can you clarify if this some additional issue fixed?